packages/core/src/physics/shape/BoxColliderShape.ts

```diff
@@ -0,0 +1,31 @@
+import { ColliderShape } from "./ColliderShape";
+import { IBoxColliderShape } from "@oasis-engine/design";
+import { Vector3 } from "@oasis-engine/math";
+import { PhysicsManager } from "../PhysicsManager";
+
+/**
+ * physical collider shape for box.
+ */
+export class BoxColliderShape extends ColliderShape {
+  private _extents: Vector3 = new Vector3(1, 1, 1);
+
+  /** extents of box shape */
+  get extents(): Vector3 {
```

Comment:
Use ‘size', 'size' is more easy to understand than `extents`  for developer.

Reply:
done